src/interfaces/new-request-interface.ts

```diff
@@ -34,7 +34,7 @@ export interface NewRequestIF {
   extendedRequestType: SelectOptionsI
   getNameReservationFailed: boolean
   helpMeChooseModalVisible: boolean
-  affiliationErrorModalVisible: boolean
+  affiliationErrorModalValue: number // 0, 1 or 2
```

Comment:
I didn't think it was worth an enum for this. Thoughts?

0 = hide modal
1 = unable to start registration
2 = associated with another account

Maybe create an enum if we add another value in the future?